scripts/nf-core

```diff
@@ -229,7 +229,7 @@ def create(name, description, author, new_version, no_git, force, outdir):
 @click.option(
     '--release',
     is_flag = True,
-    default = os.environ.get('TRAVIS_BRANCH') == 'master' and os.environ.get('TRAVIS_REPO_SLUG', '').startswith('nf-core/') and not os.environ.get('TRAVIS_REPO_SLUG', '') == 'nf-core/tools',
+    default = os.environ.get('GITHUB_REF') == 'master' and os.environ.get('GITHUB_REPOSITORY', '').startswith('nf-core/') and not os.environ.get('GITHUB_REPOSITORY', '') == 'nf-core/tools',
```

Comment:
See other comment - this will need cleaning up.

Reply:
See https://github.com/nf-core/tools/pull/502/commits/e1e8650b635951d6e9d9089cd3d73e1d39909d7a